dawn/test/integration-test/dawn4py-tests/data/generate_versioned_field_ref.cpp

```diff
@@ -37,18 +36,14 @@ class generate_versioned_field {
     static constexpr ::dawn::driver::unstructured_extent c_extent = {false, 0,0};
     static constexpr ::dawn::driver::unstructured_extent d_extent = {false, 0,0};
     static constexpr ::dawn::driver::unstructured_extent e_extent = {false, 0,0};
-    static constexpr ::dawn::driver::unstructured_extent c_0_extent = {false, 0,0};
 
     void run() {
       using ::dawn::deref;
 {
     for(int k = 0+0; k <= ( m_k_size == 0 ? 0 : (m_k_size - 1)) + 0+0; ++k) {
       for(auto const& loc : getEdges(LibTag{}, m_mesh)) {
-m_c_0(deref(LibTag{}, loc), (k + 0)) = m_c(deref(LibTag{}, loc), (k + 0));
-      }    }}{
-    for(int k = 0+0; k <= ( m_k_size == 0 ? 0 : (m_k_size - 1)) + 0+0; ++k) {
-      for(auto const& loc : getEdges(LibTag{}, m_mesh)) {
-m_a(deref(LibTag{}, loc), (k + 0)) = ((m_b(deref(LibTag{}, loc), (k + 0)) / m_c_0(deref(LibTag{}, loc), (k + 0))) + (::dawn::float_type) 5);
+m_a(deref(LibTag{}, loc), (k + 0)) = ((m_b(deref(LibTag{}, loc), (k + 0)) / m_c(deref(LibTag{}, loc), (k + 0))) + (::dawn::float_type) 5);
+      }      for(auto const& loc : getEdges(LibTag{}, m_mesh)) {
```

Comment:
This test can be removed then. It was testing that a versioned field was generated, but it's not generated anymore.